src/proto/robot_status_msg.proto

```diff
@@ -19,6 +19,7 @@ message RobotStatus
     Timestamp time_sent                               = 10;
     uint64 last_handled_primitive_set                 = 11;
     PrimitiveExecutorStatus primitive_executor_status = 12;
+    Timestamp omit_thunderloop_processing_time_sent   = 13;
```

Comment:
I see why you are doing this now -- because in Thunderscope we don't have the timestamp of the last_handled_primitive_set to do latency calculations on.

I feel that adding a new field to this proto that `last_handled_primitive_set_sent_time` helps clean up the code and logic

Reply:
I was thinking of doing the same, but after discussing with Nima, I believe it would be better to avoid adding further data to be sent over the network.

Reply:
rename in progress `adjusted_time_sent`